sqlite-db/src/delete_trivial_cfds.rs

```diff
@@ -0,0 +1,107 @@
+use crate::Connection;
+use anyhow::Result;
+
+impl Connection {
+    /// Delete all CFDs which have no events associated with them.
+    ///
+    /// It is recommended to call this method right after calling
+    /// `sqlite_db::connect`, before new CFDs are added to the
+    /// database. This is to ensure that no relevant data is deleted.
+    pub async fn delete_trivial_cfds(&self) -> Result<()> {
+        let mut conn = self.inner.acquire().await?;
+
+        // FIXME: I cannot run `cargo sqlx prepare` locally :(, so I
```

Comment:
Is this still relevant? 
Given that you don't have any compile errors I think this is outdated.
Note: I am able to run `prepare_db.sh` without problems.

Reply:
Thanks for noticing this!

You might notice that we're using `query` instead of `query!` here. `query` gives you no compile-time guarantees. I think we always want to use the macro versions of these APIs.

The tests verify that it works, but I think it would still be good to use `query!` here. Could you or someone else modify the code and run `prepare_db.sh` afterwards?

Reply:
Thanks! I'm gonna close this for now, because we don't seem to agree on it.